.github/workflows/draft-release.yml

```diff
@@ -0,0 +1,90 @@
+# Copyright 2024 The Authors (see AUTHORS file)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: 'draft-release'
+
+on:
+  workflow_dispatch:
+    inputs:
+      version_strategy:
+        description: |-
+          Update strategy
+        default: 'patch'
+        type: 'choice'
+        options:
+        - 'major'
+        - 'minor'
+        - 'patch'
+        - 'prerelease'
+        required: true
+
+env:
+  PR_BRANCH: 'automation/draft-release-${{ github.ref_name }}'
+
+jobs:
+  draft-release:
+    runs-on: 'ubuntu-latest'
+    permissions:
+      contents: 'read'
+      id-token: 'write'
+    steps:
+    - name: 'Increment version'
+      id: 'increment-version'
+      uses: 'abcxyz/pkg/.github/actions/increment-version@main' # ratchet:exclude
+      with:
+        version_strategy: '${{ inputs.version_strategy }}'
```

Comment:
You should add a VERSION file with 0.0.0 as its contents just to make sure its there for when you release.